src/constants/constantValues.ts

```diff
@@ -1,5 +1,6 @@
 export const API_KEY = process.env.REACT_APP_MOVIES_API_KEY;
 export const API_BASE_URL = process.env.REACT_APP_MOVIES_API_BASE_URL;
+export const DIRECTUS_BASE_URL = process.env.REACT_APP_MOVIES_API_DIRECTUS_URL;
```

Comment:
Directly reference **`process.env.{ENV_VARIABLE_NAME}`** in specific files. You can use the same approach as it's used in this file due to readibility.